components/sysutils/mtx/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Carsten Grzemba
+#
+BUILD_BITS= 64
+BUILD_STYLE= configure
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mtx
+COMPONENT_VERSION=	1.3.12
+COMPONENT_REVISION=	0
```

Comment:
This is not needed and we (usually) do not do that.  But since it is harmless otherwise then it could stay if preferred so.